From: uazo <uazo@users.noreply.github.com>
Date: Sat, 29 Aug 2026 13:17:39 +0000
Subject: Temp Do not compress devtools frontend resources in non-debug mode

Compressed resources currently do not work on Windows for an unknown reason.
Disable compression temporarily until the issue is understood and fixed.
---
 third_party/devtools-frontend/src/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/devtools-frontend/src/BUILD.gn b/third_party/devtools-frontend/src/BUILD.gn
--- a/third_party/devtools-frontend/src/BUILD.gn
+++ b/third_party/devtools-frontend/src/BUILD.gn
@@ -188,7 +188,7 @@ action("generate_devtools_grd") {
   ]
 
   deps = [ ":devtools_frontend_resources" ]
-  if (!is_debug) {
+  if (!is_debug && is_android) {
     deps += [ ":compress" ]
 
     # Compression is slow, particularly in debug builds, so only compress in
--
